src/video_analyze.py

```diff
@@ -844,6 +875,12 @@ def __call__(self, parser, namespace, values, option_string=None):
         "--threaded",
         action="store_true",
     )
+    parser.add_argument(
+        "--no-parse-clean",
+        action="store_true",
+        dest="no_parse_clean",
```

Comment:
Please don't use negative variables: It makes it hard to understand code (e.g. "not not not something").

Instead, have "--noparse-clean" to set "parse_clean" to False.